android/engine/src/main/java/org/smartregister/fhircore/engine/ui/pin/PinLoginActivity.kt

```diff
@@ -61,10 +64,12 @@ class PinLoginActivity : BaseMultiLanguageActivity() {
   }
 
   private fun moveToHome() {
+    syncBroadcaster.runSync()
     loginService.navigateToHome()
   }
 
   private fun moveToLoginViaUsername() {
+    Timber.i("login 2")
```

Comment:
Do we need this log? If yes, please update it to a more informative one

Reply:
@dubdabasoduba No, we don't need this log. I added it for testing. Removing it.